style.css

```diff
@@ -0,0 +1,67 @@
+body {
+  background-color: rgba(27, 27, 50, 0.8);
+}
+
+#container {
+  display: block;
+  max-width: 720px;
+  margin: 3.125rem auto 0 auto;
+}
```

Comment:
I think it would be nice to use classes instead of id attributes in your HTML file. Read _line 10_ of this [guidelines](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) to understand what I am saying.